templates/events/_resultWithoutJS.html.twig

```diff
@@ -0,0 +1,41 @@
+{% if pagination.totalResults == 1 %}
+    <h2 class="govuk-heading-m">{{ pagination.totalResults }} event(s) found</h2>
```

Comment:
Aren't these the same? Does it need an if statement?

Reply:
This was from the original file and I didn't edit anything. But I will go ahead and correct it